### DISCUSSION TITLE
How do i install a plugin?

### DISCUSSION THREAD

**Discussion by dakky** [cryinkfly/Autodesk-Fusion-360-for-Linux#448] [Q&A]
Greetings

Ich want to install this plugin: https://apps.autodesk.com/FUSION/en/Detail/Index?id=7197558650811789&os=Win64&appLang=en

How do i install the msi package?

this did not work:
```
export WINEPREFIX=~/.fusion360/wineprefixes/default
wine ~/Downloads/GridfinityGenerator_win64.msi
wine: could not open working directory L"unix\\home\\username\\.fusion360\\", starting in the Windows directory.
Application could not be started, or no application associated with the specified file.
ShellExecuteEx failed: File not found.

0c20:fixme:ver:GetCurrentPackageId (000000000021FEB0 0000000000000000): stub
```

my wine knowledge is quite limited

**Comment by hon1nbo** [comment]
I know it's late, but I found this as I was looking into it as well so figured I'd document for posterity.

You can use the supplied `bin/winetricks` script that's alongside the `autodesk_fusion_launcher.sh`

Just run it in terminal and a window will pop up. You can select to run an Arbitrary Executable (exe/msi/etc). Winetricks will prompt for some privacy settings if you haven't run it interactively before.
With that you can select the MSI for the plugin, then once it's installed you can reboot Fusion and it should be there.

Oddly the `WINEPREFIX` is actually correct in your post, and it's also the very first thing I did as well. I'm not sure right away why it doesn't work out of the gate like that, my guess is there's some optimization getting in the way. Since I'm focused on getting my CAD drawings exported to my Shaper, I'll leave the debugging for another time and just document this for now.

EDIT: There's also this, which is basically identical to another method I tried already but is supposed to be the method that works: https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/wiki/Documentation#install-a-extension

~H
